net-vpn/candy/candy-5.10.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake systemd readme.gentoo-r1
+
+DESCRIPTION="A reliable, low-latency, and anti-censorship virtual private network"
+HOMEPAGE="https://github.com/lanthora/candy"
+SRC_URI="https://github.com/lanthora/candy/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~mips"
+
+DEPEND="
+	dev-libs/poco
+	dev-libs/openssl
+	dev-libs/spdlog
+"
+RDEPEND="
+	${DEPEND}
+	dev-libs/libfmt
+"
+
+src_install(){
+	cmake_src_install
+	default
+
+	insinto /etc
+	doins candy.cfg
+	fperms 0644 /etc/candy.cfg
```

Comment:
this is not necessary, the default permission is 0644, please check
or do you want to make it more secure, like set to 0600?